README.md

```diff
@@ -1,5 +1,63 @@
 # RSK Next JS Box
 
+This box comes with everything you need to start using smart contracts from a react app and Next JS on RSK Network.
+
+## Installation
+
+First ensure you are in a new and empty directory.
+
+1. Run the `unbox` command via `npx` and skip to step 3. This will install all necessary dependencies. A Create-React-Express-App is generated in the `app` directory.
```

Comment:
It is not clear why a step 2 then

Reply:
This is the way that truffle uses to explain how to install the box, look this:
https://www.trufflesuite.com/boxes/react.
But beside that, I found an issue, I will change Create-React-Express-App to Creat-React-Next-App.

Reply:
@alepc253 you can use the command `npx truffle unbox rsksmart/rsk-next-box` to install the box. But if you prefer use `truffle unbox rsksmart/rsk-next-box` you need to have installed truffle in your machine.